primenet.py

```diff
@@ -6763,7 +6765,7 @@ def is_pyinstaller():
     )
 
 if options.min_exp < 1000000000 < options.max_exp:
```

Comment:
These both default to None, so you would need to check if they are truthy first:
```py
if options.min_exp and options.max_exp and options.min_exp < 1000000000 <= options.max_exp:
```